src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.tsx

```diff
@@ -2,17 +2,14 @@ import React from 'react';
 import { css, cx } from 'emotion';
 
 import { palette } from '@guardian/src-foundations';
-import { border } from '@guardian/src-foundations/palette';
 import { headline } from '@guardian/src-foundations/typography';
 import { from, between, Breakpoint } from '@guardian/src-foundations/mq';
 
-import { useApi } from '@root/src/web/lib/api';
 import { namedAdSlotParameters } from '@root/src/model/advertisement';
 import { AdSlot, labelStyles } from '@root/src/web/components/AdSlot';
+import { Lazy } from '@root/src/web/components/Lazy';
 
-import { joinUrl } from '@root/src/web/lib/joinUrl';
-import { MostViewedFooterGrid } from './MostViewedFooterGrid';
-import { SecondTierItem } from './SecondTierItem';
+import { MostViewedFooterData } from './MostViewedFooterData';
```

Comment:
By creating a separation between the section layout and the data fetching it lets us render the advert in the container component and then wrap the data fetching logic inside `Lazy`